src/components/markdown/markdown-parser.ts

```diff
@@ -42,7 +42,7 @@ export async function markdownToHTML(
             // Allow the `style` attribute on all elements
             attributes: {
                 ...defaultSchema.attributes,
-                '*': ['style'],
+                '*': ['height', 'style', 'width'],
```

Comment:
Here's the fixup @Kiarokh. Since the fix was in a separate file, and the whole PR is so small, I squashed and force-pushed, rather than pushing a separate fixup.